targets/TARGET_Realtek/TARGET_AMEBA/TARGET_RTL8195A/device/TOOLCHAIN_ARM_STD/rtl8195a.sct

```diff
@@ -18,29 +18,50 @@ LR_IRAM 0x10007000 (0x70000 - 0x7000) {
   }
 
   ER_IRAM +0 FIXED {
-    *rtl8195a_crypto.o(.text*, .rodata*)
-    *mbedtls*.o(.text*, .rodata*)
-    libc.a: (.text*, .rodata*)
+    *rtl8195a_crypto.o (+RO)
+    * (i.mbedtls*)
+    *libc.a (+RO)
+    
+	*rtx_*.o (+RO)
```

Comment:
We find mbed is very sensitive to timing drift, so we put ticker related code to SRAM.